src/Security/Security.php

```diff
@@ -397,18 +460,15 @@ public function LoginForm()
      * @return array Returns an array of available login forms (array of Form
      *               objects).
      *
-     * @todo Check how to activate/deactivate authentication methods
      */
-    public function GetLoginForms()
+    public function getLoginForms()
```

Comment:
Not sure, I think getLoginForms is now delegated to delegateToHandlers (or whatever we called it). That builds a tabbed group of forms now.